presto-docs/src/main/sphinx/release/release-0.131.rst

```diff
@@ -0,0 +1,8 @@
+=============
+Release 0.131
+=============
+
+General Changes
+---------------
+
+* Fix poor performance of redistributing dictionary encoded data.
```

Comment:
"Fix poor performance of transporting dictionary encoded data over the network." might be more clear